src/installer/dbinstall_mysql.sql

```diff
@@ -36,6 +36,7 @@ INSERT INTO `DBPREFIXconfig` (`identifier`, `type`, `value`, `user_editable`) VA
 ('adminstatus_offlinehiddenbydefault', 'BOOL', 'false', 1),
 ('imprint_enabled', 'BOOL', 'false', 1),
 ('imprint_url', 'STRING', 'imprint.php', 1),
+('assigner_cooldown_period', 'INT', '0', 1),
```

Comment:
Value is still `assigner_cooldown_period`, but `Assigner:getCooldownPeriod` returns `assigner_cooldown_seconds`.